data/mods/gen9ssb/moves.ts

```diff
@@ -2554,6 +2554,36 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Ice",
 	},
 
+	// YveltalNL
+	highground: {
+		accuracy: 100,
+		basePower: 90,
+		category: "Special",
+		shortDesc: "If user is taller than the opponent, boosts Sp. Atk by 1 stage.",
+		name: "High Ground",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		volatileStatus: "highground",
+		condition: {
+			onStart(target, source, sourceEffect) {
+				if (source.getHeight() > target.getHeight()) {
+					this.boost({spa: 1}, source);
+				}
+			},
+		},
```

Comment:
a volatile is the wrong thing to use here

Reply:
changing to onHit, reverting the /sim changes in a bit too. tbf it is odd how sim has no way to get height already, even if no mechanic uses it